src/code/utils/codap-connect.coffee

```diff
@@ -0,0 +1,49 @@
+IframePhoneRpcEndpoint = require '../../vendor/iframe-phone-rpc-endpoint'
+
+module.exports = class CodapConnect
+
+  name: 'Building Models Tool'
+
+  codapPhone: null
+
+  initAccomplished: false
+
+  init: (linkManager, name) ->
+    log.info 'CodapConnect initializing'
+    @linkManager = linkManager
+    name and @name = name
+
+    @codapPhone = new IframePhoneRpcEndpoint( @codapRequestHandler,
+      'codap-game', window.parent )
+
+    @codapPhone.call( {
+      action: 'initGame'
+      args:
+        name: @name
+        dimensions:
+          width: 800
+          height: 600
+        contextType: 'DG.DataContext'
+    }, @initGameHandler)
+
+  codapRequestHandler: (iCmd, iCallback) =>
```

Comment:
Are most codap interactions handled this way, where Codap initiates the calls? Are there ways for Building models to make calls into Codap?